org.opent2t.onboarding.winkhub/js/README.md

```diff
@@ -0,0 +1,5 @@
+# Wink Hub Onboarding
+Sample [Wink Hub](http://www.wink.com/) onboarding module for Open Translators to Things. May be used to discover devices connected to a Wink Hub per the 
+[org.OpenT2T.Onboarding.WinkHub.xml](https://github.com/openT2T/onboarding/blob/master/org.OpenT2T.Onboarding.WinkHub/org.OpenT2T.Onboarding.WinkHub.xml) schema.
```

Comment:
>org.OpenT2T.Onboarding.WinkHub.xml [](start = 1, length = 34)

Not sure if it matters, but since you renamed the manifest to all lower-case, do you want to update this readme to match?